packages/subgraph/src/v1-analytics/mapping.ts

```diff
@@ -291,8 +332,52 @@ export function handleTransactionCancelled(event: TransactionCancelled): void {
       assetBalance.assetId = transaction.receivingAssetId;
       assetBalance.router = event.params.router.toHex();
       assetBalance.amount = new BigInt(0);
+      assetBalance.supplied = new BigInt(0);
+      assetBalance.locked = new BigInt(0);
     }
     assetBalance.amount = assetBalance.amount.plus(transaction.amount);
+    assetBalance.locked = assetBalance.locked.sub(transaction.amount);
     assetBalance.save();
   }
+
+  // update metrics
+  let timestamp = event.block.timestamp.toI32();
+
+  let hour = timestamp / 3600; // rounded
+  let hourStartTimestamp = hour * 3600;
```

Comment:
lol isn't this the same value as `timestamp`

Reply:
yes

Reply:
jk i was wrong, wang was right. rounding makes them different

Reply:
apparently it is not, the hour value is rounded